scripts/gen/tpcc.sh

```diff
@@ -116,232 +75,32 @@ exec &> >(tee -a "$logdir/script.log")
-    then
-      echo "Decreasing upper limit because of machine type $machinetype."
-      ((f_active-=500))
-      ((f_warehouses-=1700))
-    fi
-  fi
-fi
-
 if [ -z "$f_skip_load" ]
 then
-  echo "configuring the cluster for fast import..."
-  ./cockroach sql --insecure --url "${pgurls[0]}" -e "
-  SET CLUSTER SETTING kv.bulk_ingest.max_index_buffer_size = '2gib';
-  SET CLUSTER SETTING kv.bulk_io_write.concurrent_addsstable_requests = 10;
-  SET CLUSTER SETTING schemachanger.backfiller.max_buffer_size = '5GiB';
-  SET CLUSTER SETTING kv.snapshot_recovery.max_rate = '128 MiB';
-  SET CLUSTER SETTING kv.snapshot_rebalance.max_rate = '128 MiB';
-  ";
-  echo "importing ..."
-  ./cockroach workload fixtures import tpcc --warehouses="$f_warehouses" --active-warehouses="$f_active" "${pgurls[0]}"
-  echo "done importing"
-fi
-
-# Comment out since we don't have a scenario using this at present.
-## if f_inc is not specified, then we set it equal to f_warehouses, this means
-## we just need to run one iteration.
-#if [[ $f_inc == 0 ]];
-#then
-#  f_inc=$f_warehouses
-#fi
-
-# We first run an exploration test round with binary search to find a right
-# range of test boundary to use in the subsequent official run.
-# For certain machine types the binary search result don't work well and will
-# cause kv errors and end the test, so we need to handle them differently.
-tpcc_files=""
-f_inc=50
-has_good_result=0
-if [[ $special_type -eq 0 ]];
-then
-  l=$f_active
-  r=$f_warehouses
-  while [[ $l -le $r ]]
-  do
-      mid=$((l + ((r - l)/2)))
-      report="${logdir}/tpcc-results-$mid.txt"
-      ./cockroach workload run tpcc \
-          --warehouses="$mid"  \
-          --active-warehouses="$mid" \
-          --ramp=1m --duration="1m" \
-          "${pgurls[@]}" > "$report"
-
-      # awk uses lexicographic order by default when doing comparison, we
-      # need to force it to use number comparison.
-      prev_line=$(tail -2 "$report")
-      if [[ "$prev_line" == *efc* ]] && [[ $(tail -1 "$report" | awk '{if(int($3) > 87){print "pass"}}') == "pass" ]];
-      then
-        echo "Test TPCC on $mid failed"
-        ((r=mid-f_inc))
-        current_result_good=0
-      else
-        echo "Test TPCC on $mid passed"
-        ((l=mid+f_inc))
-        current_result_good=1
-        if [[ $mid -gt $best_result ]];
-        then
-          best_result=$mid
-        fi
-      fi
-  done
+  #./cockroach sql --insecure --url "${pgurls[0]}" -e "
+  # SET CLUSTER SETTING kv.snapshot_recovery.max_rate = '512 MiB';
+  # SET CLUSTER SETTING kv.snapshot_rebalance.max_rate = '512 MiB';
+	# SET CLUSTER SETTING admission.kv.enabled,=false;
+	# SET CLUSTER SETTING admission.sql_kv_response.enabled=false;
+	# SET CLUSTER SETTING admission.sql_sql_response.enabled=false;
+  #";
+  echo "Loading TPCC fixture for $f_warehouses warehouses ..."
+  ./cockroach workload fixtures load tpcc --checks=false --warehouses="$f_warehouses" "$f_load_args" "${pgurls[0]}"
```

Comment:
quote around `$f_load_args` should be removed (otherwise the whole line will be interpreted as something like
```
./cockroach workload fixtures load tpcc --checks=false --warehouses=900 '--provider-override=gs --bucket-override=cloud-report-tpcc' 'postgres://root@xx.xx.xx.xx:26257?sslmode=disable'
```

Reply:
Done.